### DISCUSSION TITLE
Best way to deal with cmake find_package?

### DISCUSSION THREAD

**Discussion by xumykargo** [bazelbuild/rules_foreign_cc#1042] [Q&A]
Hi,

I am building some external project depending on opencv. The project uses cmake find_package to find opencv. I have built opencv from source with bazel and it's available as a target, but what's the best way to make find_package find that target?

Also on my machine there's a system installed opencv, and how do I ensure hermetic build by preventing cmake to find the system opencv? Does sandbox work here? Asking because the cmake rule seems to use the system opencv.

Thanks!

**Comment by Jark5455** [comment]
Have you tried adding opencv in bazel and then adding `deps = ["@opencv"]` to the cmake rule?
